fcl/protodunevd/detsim/protodunevd_refactored_detsim_pureAr.fcl

```diff
@@ -0,0 +1,70 @@
+#include "services_refactored_pdune.fcl"
```

Comment:
This fhicl could be much simplified by inheriting from the one with Xe. Something like:
```
#include "protodunevd_refactored_detsim_Xe10ppm.fcl"
physics.producers.sipmAr:    @local::xarapuca_ar
physics.producers.opdigi:    @local::standard_vddaphne
physics.simulate: [ rns,  
              # TPC simulation
              tpcrawdecoder,
              # OpDet and CRT simulation
              sipmAr, opdigi
              # crt
               ] 
```